addons/studio-invite-button/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ addon, global, console }) {
+// RegExp to test for a valid url and get the studio id
+const urlRegex = /^https?:\/\/scratch\.mit\.edu\/studios\/(\d+)\/comments($|\/)/;
```

Comment:
> Change the match patterns, don't use a regex

In this case they're testing to see if the user is on comments, but there should be a better way to do that

Reply:
> In this case they're testing to see if the user is on comments, but there should be a better way to do that

For now doing it using a match pattern is fine because studio tabs are handled server-side, but when the studio page is migrated to `scratch-www` the best way will be `waitForElement`.